pkg/kinflate/resource/appresource.go

```diff
@@ -39,3 +39,15 @@ func NewFromPath(path string, loader loader.Loader) ([]*Resource, error) {
 	}
 	return res, nil
 }
+
+func NewFromPathList(pathList []string, loader loader.Loader) ([]*Resource, error) {
```

Comment:
Should we export both this one and the other one?

Reply:
Or should we move this one somewhere else?

Reply:
I guess expose only one is fine.

Reply:
If we keep one, then maybe the signature should be:
`func NewFromPaths(loader loader.Loader, paths ...string) ([]*Resource, error)`